ocis-pkg/handlers/checktcp.go

```diff
@@ -0,0 +1,23 @@
+package handlers
+
+import (
+	"context"
+	"net"
+)
+
+// NewTCPCheck returns a check that connects to a given tcp endpoint.
+func NewTCPCheck(address string) func(ctx context.Context) error {
+	return func(_ context.Context) error {
+		conn, err := net.Dial("tcp", address)
```

Comment:
Does it make sense to set the timeout?

Reply:
@2403905 good point.... will look into it